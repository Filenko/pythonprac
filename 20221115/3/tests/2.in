try:
	alp = Alpha(pp=3, c=10, z=2, a=42)
except AttributeError:
	print('ОШИБКА')

alp = Alpha(c=5, z=4, b=8)
alp.e = 123
print(alp)
alq = AlphaQ(c=10, z=2, a=42)
alq.e = 123
print(alq)
