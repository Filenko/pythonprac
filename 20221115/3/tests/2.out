ОШИБКА
b: 8, c: 5, e: 123, z: 4
a: 42, c: 10, e: 123, z: 2
